diff --git a/node_modules/@oraichain/tonbridge-utils/src/blockchain/BlockUtils.js b/node_modules/@oraichain/tonbridge-utils/src/blockchain/BlockUtils.js
index e9f60bc..fb6559c 100644
--- a/node_modules/@oraichain/tonbridge-utils/src/blockchain/BlockUtils.js
+++ b/node_modules/@oraichain/tonbridge-utils/src/blockchain/BlockUtils.js
@@ -136,7 +136,7 @@ export function loadBits(cell, t, n) {
     if (t.cs + n > cell.bits.cursor) {
 
         // throw Error("cannot load bits");
-        console.log("cannot load bits");
+        // console.log("cannot load bits");
         return 0;
     }
 
